src/to-have-form-values.js

```diff
@@ -0,0 +1,130 @@
+import {matcherHint} from 'jest-matcher-utils'
+import jestDiff from 'jest-diff'
+import {checkHtmlElement} from './utils'
+
+function uniq(value, index, self) {
+  return self.indexOf(value) === index
+}
+
+function isEqual(a, b) {
```

Comment:
Maybe we should just use `lodash.isequal`

Reply:
Yes, I initially resisted to add lodash as a dependency, but as I needed more and more, I considered it.

Reply:
I often resist installing lodash, but then I remind myself that the chances someone doesn't already have lodash installed in their `node_modules` are very very slim 😅

Reply:
However, there's still something else being done here besides a mere `isEqual`. Because we need to compare arrays as sets, not as arrays. Order is not important. But anyway, lodash will surely help with that anyway. This was indeed a sort of hack to get this going.